src/app/zap-templates/zcl/data-model/chip/general-diagnostics-cluster.xml

```diff
@@ -53,6 +53,23 @@ limitations under the License.
     <item name="SoftwareUpdateCompleted" value="0x05"/>
     <item name="SoftwareReset" value="0x06"/>
   </enum>  
+  <enum name="InterfaceType" type="ENUM8">
+    <item name="Unspecified" value="0x00"/>
+    <item name="WiFi" value="0x01"/>
+    <item name="Ethernet" value="0x02"/>
+    <item name="Cellular" value="0x03"/>
+    <item name="Thread" value="0x04"/>
+  </enum>
+  <struct name="NetworkInterfaceType">
```

Comment:
This one is weird; in the spec it's just "NetworkInterface", so not sure why "Type" is added here.  Presumably this is just copy/paste, but maybe a followup to fix that?

Reply:
Will do a followup. I guess the name can be updated either when `CHAR_STRING` is implemented for struct or when `HWADR` type is implemented.